libdnf/rpm/rpm_log_guard.cpp

```diff
@@ -76,4 +75,18 @@ RpmLogGuard::~RpmLogGuard() {
 }
 
 
+static int rpmlog_callback_strings(rpmlogRec rec, rpmlogCallbackData data) {
+    std::string msg(rpmlogRecMessage(rec));
+    if (!msg.empty() and msg[msg.length() - 1] == '\n') {
```

Comment:
Yes C++ supports the `and` operator, but I prefer to use one type of syntax (`&&`) in a project.

Reply:
Sorry, that's my Python background seeping through. Fixed.